test-crates/tests/build.rs

```diff
@@ -1,20 +1,23 @@
-use std::path::Path;
-
 fn main() {
     let out_dir = std::env::var_os("OUT_DIR").unwrap();
 
     println!("cargo:rerun-if-changed=*");
 
-    for &m in ["test_wasm", "wasm_memory", "wasm_empty", "wasm_io"].iter() {
-        let cargo_toml = Path::new(m).join("Cargo.toml");
-
+    for &m in [
+        "test_wasm_core",
+        "test_wasm_memory",
+        "test_wasm_empty",
+        "test_wasm_io",
+    ]
+    .iter()
+    {
         let cargo_command = std::env::var_os("CARGO");
         let cargo_command = cargo_command.as_deref().unwrap_or_else(|| "cargo".as_ref());
 
         let status = std::process::Command::new(cargo_command)
             .arg("build")
-            .arg("--manifest-path")
-            .arg(cargo_toml)
+            .arg("-p")
```

Comment:
specify the package by name since it is a member of the root workspace, rather than specifying a path to its manifest

Reply:
Yes, valid change. 

In build scripts (and in Bash scripts too) I prefer long flag names because it's easier to track down if something changes. Probably unlikely for cargo to change what `-p` means but I'd still prefer `--package` just in case. In my mind, short flags are for human users running a command once on the command line and long flags are good for automation and long term comprehension

Reply:
done